torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -61,6 +67,12 @@ def horizontal_flip_image(image: torch.Tensor) -> torch.Tensor:
 def _horizontal_flip_image_pil(image: PIL.Image.Image) -> PIL.Image.Image:
     return _FP.hflip(image)
 
+def _horizontal_flip_image_cvcuda(image: "cvcuda.Tensor") -> "cvcuda.Tensor":
```

Comment:
Should we expose this in "vision/torchvision/transforms/v2/functional/__init__.py"?